gcc/testsuite/gcc.target/riscv/vector-abi-2.c

```diff
@@ -0,0 +1,14 @@
+/* { dg-do compile } */
+/* { dg-options "-march=rv64gcv -mabi=lp64d" } */
+
+#include "riscv_vector.h"
+
+vint32m1_t
+fun (vint32m1_t* a) {  return *a; }  /* { dg-warning "the vector type" } */
```

Comment:
Could you add a test that pass or return pointer of vector type only, and make sure their won't have warning on that case?

Reply:
Ok